nexis/lib/widgets/close_button.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
```

Comment:
Should probably make a general "IconButton" widget and let people choose what Icon to use in it. Or find some Material library

Reply:
this is also tmeporary, i will remove it later, but need it as it is for now.